src/org/daisy/dotify/formatter/impl/segment/ExternalReferenceSegment.java

```diff
@@ -0,0 +1,39 @@
+package org.daisy.dotify.formatter.impl.segment;
+
+/**
+ * This segment contains all attributes from the optional
+ * metadata tags that could be added to blocks in order to
```

Comment:
The "metadata" here is probably a remainder of your first version.

Reply:
You are right.